src/Engine/ProtoCore/Properties/Resources.en-US.resx

```diff
@@ -921,7 +921,7 @@ Parameter name: {0}. href=ArgumentNullException.html</value>
     <value>Invalid list@level syntax, consider using @L1 for level 1.</value>
   </data>
   <data name="VariableRedifinitionError" xml:space="preserve">
-    <value>Variable {0} has been defined in this code block node. You cannot define a variable more than once. href=testing.html</value>
+    <value>Variable {0} has been defined in this code block node. You cannot define a variable more than once.</value>
```

Comment:
I don't see this value changed in the Resources.resx. Can you check that.

Reply:
Resources.resx didn't have the error so it wasn't necessary to correct.

Reply:
It does have the variable in Resources.resx, but it was already correct 🙂 (didn't have the href=testing.html link)